quantities/measures/Map.cs

```diff
@@ -1,20 +1,25 @@
+using Quantities.Numerics;
+
 namespace Quantities.Measures;
 
 internal delegate Double Scale(in Double value);
 
 internal sealed class Map
```

Comment:
This also simplifies the implementation of the ´Map´ class, as all it requires is a polynomial.